Mutagen.Bethesda.Analyzers.Skyrim/Extensions/WorldspaceExtension.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Mutagen.Bethesda.Plugins.Cache;
+using Mutagen.Bethesda.Skyrim;
+namespace Mutagen.Bethesda.Analyzers.Skyrim;
+
+public static class WorldspaceExtension
+{
+    public static IEnumerable<ILocationGetter> GetWorldLocations(this IWorldspaceGetter world, ILinkCache linkCache)
+    {
+        var worldLocations = new HashSet<ILocationGetter>();
```

Comment:
Either will want to be `List<ILocationGetter>` or `HashSet<IFormLinkGetter<ILocationGetter>>`

This right now is gonna has all the fields of the location, which is not desired, i dont think